Documentation/Diagnostics/PH2117.md

```diff
@@ -0,0 +1,53 @@
+# PH2117: Avoid Unnecessary Where()
+
+| Property | Value  |
+|--|--|
+| Package | [Philips.CodeAnalysis.MaintainabilityAnalyzers](https://www.nuget.org/packages/Philips.CodeAnalysis.MaintainabilityAnalyzers) |
+| Diagnostic ID | PH2117 |
+| Category  | [Maintainability](../Maintainability.md) |
+| Analyzer | [AvoidUnnecessaryWhereAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidUnnecessaryWhereAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
```

Comment:
We should update the code to remove the helpUri (of everything I guess), so that the auto-generated HelpUri will point here.

Reply:
True, I plan to clean all these up in a separate PR